modules/core/src/main/java/org/apache/ignite/internal/processors/job/GridJobWorker.java

```diff
@@ -484,7 +483,8 @@ boolean initialize(GridDeployment dep, Class<?> taskCls) {
             job = SecurityUtils.sandboxedProxy(ctx, ComputeJob.class, job);
         }
         catch (IgniteCheckedException e) {
-            U.error(log, "Failed to initialize job [jobId=" + ses.getJobId() + ", ses=" + ses + ']', e);
+            if (log.isDebugEnabled())
+                U.error(log, "Failed to initialize job [jobId=" + ses.getJobId() + ", ses=" + ses + ']', e);
```

Comment:
I think we should use the debug level here and in all other changed places.

Reply:
Besides LGTM.

Reply:
There is no implementation of `log.debug` that accepts exceptions. Is there a good way to print stacktraces to log in debug level?

Reply:
Makes sense. Let's keep it as for now .